common4j/src/main/com/microsoft/identity/common/java/exception/ErrorStrings.java

```diff
@@ -457,6 +457,9 @@ private ErrorStrings() {
                     "Please make sure to use your organizational account. " +
                     "If that doesn’t help, please return the device to your administrator.";
 
+    public static final String MISSING_SLK_ERROR_MESSAGE =
```

Comment:
If SLK was never passed then I believe it simply means this was not an Account Transfer request and rather was an AcquireTokenInteractive request? (because IIRC, you were using the present of slk to determine that we are in AT flow)

Reply:
> This is no longer the case with the new api path, there will be a new command parameters class for oneauth to use specifically for ATv2

Interesting. So sounds like we ultimately decided to create a new API for this as opposed to using ATI. Can we point me to this new API? I didn't see it in this PR

Reply:
It's coming in another PR, I did not combine them because the change list would be pretty large

Reply:
The API PRs are still in draft, i'm waiting to merge these in to then publish the other ones, as the API PRs are dependent on some things i added in these 2 PRs